livechat/configuration/api/v35.py

```diff
@@ -1,5 +1,7 @@
 ''' Configuration API module with client class in version 3.5. '''
 
+from typing import List
```

Comment:
Considering adding such type hint, I'm wondering whether we should change already existing `arg: list` entries accordingly.

Reply:
replied on slack; tl;dr: let's use the more verbose type hints, but add them incrementally, as updating it everywhere would vastly expand the scope of this task